src/components/transactions/GovDelegation/GovDelegationModalContent.tsx

```diff
@@ -34,32 +36,48 @@ export enum ErrorType {
   NOT_AN_ADDRESS,
 }
 
-export const GovDelegationModalContent = () => {
-  const { chainId: connectedChainId, readOnlyModeAddress } = useWeb3Context();
+type GovDelegationModalContentProps = {
+  type: ModalType.RevokeGovDelegation | ModalType.GovDelegation;
+};
+
+export const GovDelegationModalContent: React.FC<GovDelegationModalContentProps> = ({ type }) => {
```

Comment:
I'm not sure why, but there's some styling differences for this component when shown for revoke and delegation

Reply:
Oh I see, it's when some of the toggles are disabled, the styling is different. I think @MareskoY comment makes sense, to not show the selector if only one option is going to be enabled.